astropy/uncertainty/core.py

```diff
@@ -32,9 +43,13 @@ class Distribution:
     Parameters
     ----------
     samples : array-like
-        The distribution, with sampling along the *leading* axis. If 1D, the
-        sole dimension is used as the sampling axis (i.e., it is a scalar
-        distribution).
+
+    The distribution, with sampling along the *trailing* axis. If 1D, the sole
+    dimension is used as the sampling axis (i.e., it is a scalar distribution).
+    If an |ndarray| or subclass, the data will not be copied unless it is not
+    possible to take a view (generally, only when the strides of the last axis
+    are negative).
```

Comment:
```suggestion

        The distribution, with sampling along the *trailing* axis. If 1D, the sole
        dimension is used as the sampling axis (i.e., it is a scalar distribution).
        If an |ndarray| or subclass, the data will not be copied unless it is not
        possible to take a view (generally, only when the strides of the last axis
        are negative).
```
Not sure if this is strictly necessary (it was at one point but maybe it's changed?), but most of the other docstrings follow the indented-under-parameters convention

Reply:
Oops, no, that is me trying to get line links in other, forgetting to remove a blank line, and then some pre-commit changing the indent. Thanks for noticing! Now corrected.